src/OSPSuite.Core/OSPSuite.Core.csproj

```diff
@@ -59,24 +59,24 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="MathNet.Numerics, Version=3.20.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\MathNet.Numerics.3.20.0\lib\net40\MathNet.Numerics.dll</HintPath>
+    <Reference Include="MathNet.Numerics, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Also updated packages that were outdated. Since we are using tests for all of them, I do not expect any big surprise
Good news is that all those new packages support .NET Standard